garden-backend-service/src/api/schemas/mdf_dataset.py

```diff
@@ -0,0 +1,13 @@
+from pydantic import Field
+
+from .base import BaseSchema, UniqueList
+
+
+class DatasetMetadata(BaseSchema):
```

Comment:
this might get confusing since we already have a `_DatasetMetadata` schema (only used by entrypoints so far) , so let's disambiguate 
```suggestion
class MDFDatasetMetadata(BaseSchema):
```